src/main/cpp/mechanisms/noteManager/decoratormods/PassState.cpp

```diff
@@ -72,10 +72,7 @@ bool PassState::AtTarget()
 	double botSpeed = units::angular_velocity::radians_per_second_t(units::angular_velocity::revolutions_per_minute_t(m_mechanism->getlauncherBottom()->GetRPS() * 60)).to<double>();
 	double targetSpeed = m_mechanism->getlauncherTargetSpeed().value();
 
-	bool topSpeedIsWithinTolerance = topSpeed > (targetSpeed * 0.95);
-	bool bottomSpeedIsWithinTolerance = botSpeed > (targetSpeed * 0.95);
-
-	attarget = angleIsWithinTolerance; //&& topSpeedIsWithinTolerance && bottomSpeedIsWithinTolerance;
+	attarget = angleIsWithinTolerance;
```

Comment:
It is fine to delete it here, but I have kept it in my pass updated branch and just commented it out. Beucase if we do get the speed changing working we want to keep this

Reply:
Yeah, I figured that, but we can always go back to this point and find the code to add back in.  I was trying to get rid of the build warnings.